### ISSUE TITLE
Add JuliaMono as default font in VSCode

### ISSUE THREAD

**Issue by heetbeet** [heetbeet/juliawin#50]
The idea is to download JuliaMono from: https://github.com/cormullion/juliamono/blob/master/juliamono-ttf.zip and add the `.ttf`s to `packages\fonts`.

Then before any VSCode launch, the following code can be adapted and run to ensure all fonts are temporary available to the system https://gist.github.com/Jaykul/d53a16ce5e7d50b13530acb4f98aaabd , for eg `bin/activate-juliawin-fonts.ps1` with content:
```
add-type -name Session -namespace "" -member @"
[DllImport("gdi32.dll")]
public static extern int AddFontResource(string filePath);
"@

$null = foreach($font in Get-ChildItem -Path "$juliawin_packages\fonts" -Recurse -Include *.ttf, *.otg) {
    [Session]::AddFontResource($font.FullName)
}
```

On the VSCode side we should add JuliaMono in the font setting, (over here, https://github.com/heetbeet/juliawin/blob/main/userdata/.vscode/data/user-data/User/settings.json ), similarly to this guide https://dev.to/thegeoffstevens/vs-code-settings-you-should-customize-5e75 but replacing Fira with Juliamono:
```
"editor.fontFamily": "Fira Code"
"editor.fontLigatures": true
```

**Comment by RoyiAvital** [comment]
Does it mean the Font will be added only for the session of VS Code?
What happens if VS Code crashes?
Or is it added for as long as the kernel (Windows Kernel) session is a live?

**Comment by heetbeet** [comment]
This `.ps1` script allows the fonts to be added to Windows's font cache without installing it. It is accessible by any program until the next reboot (for the duration of the current Windows Kernel). 

Ideally I would have have wanted the font to only be active for the duration of the VSCode session and only accessible to VSCode. This, however, it not possible as far as I know.

**Comment by heetbeet** [comment]
Hmm it seems like the font is added _permanantly_ to the user's fonts. This is a bit rude and not ideal.

**Comment by heetbeet** [comment]
[activate-juliamono-font.zip](https://github.com/heetbeet/juliawin/files/9653210/activate-juliamono-font.zip)

**Comment by RoyiAvital** [comment]
Is the attachment a solution for this or the permanent addition?

**Comment by heetbeet** [comment]
The attachment is a solution for portably adding the fonts to the current windows session. I'm still not sure if it's a good idea since every Windows app will reflect the fonts until the next reboot.

**Comment by RoyiAvital** [comment]
Maybe have a documented option for Opt In?

**Comment by heetbeet** [comment]
Yes, like maybe an env variable (that can now be set in startup.cmd). 
Like `set JULIAWIN_JULIAMONO_FONTS=true`

**Comment by RoyiAvital** [comment]
Yep, that would be great. Once it is there, let me know, I will document it in the Wiki page.

**Comment by RoyiAvital** [comment]
Any chance to have this in the `1.9.x` version?
